user-interface/src/main/java/life/qbic/datamanager/views/general/DialogWindow.java

```diff
@@ -29,6 +33,13 @@ protected DialogWindow() {
     confirmButton.addClickListener(this::onConfirmClicked);
   }
 
+  protected void initCancelShortcuts(Command onCreationCanceled) {
+      setCloseOnOutsideClick(false);
+      setCloseOnEsc(false);
+      Shortcuts.addShortcutListener((Component) this,
```

Comment:
This casting seems redundant
```suggestion
      Shortcuts.addShortcutListener( this,
```